src/PrestaShopBundle/Resources/views/Admin/Sell/Order/Order/Blocks/View/payments.html.twig

```diff
@@ -38,6 +38,22 @@
           <strong>{{ orderForViewing.payments.paidAmount }}</strong>
           {{ 'paid instead of'|trans({}, 'Admin.Orderscustomers.Notification') }}
           <strong>{{ orderForViewing.payments.amountToPay }}</strong>
+
+          {% if orderForViewing.brothers.brothers is not empty %}
+            
+            {% set firstBrother = orderForViewing.brothers.brothers|first %}
+            {% if orderForViewing.brothers.brothers | length == 1 %}
+              <br/>{{ 'This warning also concerns order '|trans({}, 'Admin.Orderscustomers.Notification') }}
+            {% else %}
+              <br/>{{ 'This warning also concerns the next orders:'|trans({}, 'Admin.Orderscustomers.Notification') }}
+            {% endif %}
+
+            {% for brother in orderForViewing.brothers.brothers %}
+              <a class="_blank" href="{{ path('admin_orders_view', {'orderId': brother.orderId}) }}">
```

Comment:
Hi @matks @PierreRambaud 

Currently both `class="_blank"` and `target="_blank"` are proper. This is because https://github.com/PrestaShop/PrestaShop/blob/develop/js/tools.js#L761

Can you check https://github.com/PrestaShop/PrestaShop/pull/19710 please ?